Sources/AblyAssetTrackingPublisher/Services/DefaultLocationService.swift

```diff
@@ -12,19 +12,43 @@ class DefaultLocationService: LocationService {
 
     init(mapboxConfiguration: MapboxConfiguration,
          historyLocation: [CLLocation]?,
-         logHandler: LogHandler?) {
-        
+         logHandler: LogHandler?,
+         vehicleProfile: VehicleProfile) {
+
         let directions = Directions(credentials: mapboxConfiguration.getCredentials())
-        NavigationSettings.shared.initialize(directions: directions, tileStoreConfiguration: .default)
-        
+
+        NavigationSettings.shared.initialize(directions: directions, // or .shared
+                                             tileStoreConfiguration: .default, // or .default
```

Comment:
Hmm, I'm a bit confused about what those 2 above comments mean to be honest.

Reply:
Good catch, they were leftover from copy/paste code :) a0f9cb797eef475333d1c1a0955266e622d73eca

Reply:
Resolving on behalf of Jakub